config/dev.config

```diff
@@ -0,0 +1,105 @@
+[
```

Comment:
I see changes in this file compared to sys.config are:

```
@@ -23,16 +21,11 @@
                         {listen_address, <<"0.0.0.0">>}
                       ]}
           ]},
-      {enable_debug_endpoints, false} %% CAUTION: Debug endpoints may be inefficient
+      {enable_debug_endpoints, true} %% CAUTION: Debug endpoints may be inefficient
   ]},

-  {aecore, [
-      {peers, [
-        <<"aenode://pp$QU9CvhAQH56a2kA15tCnWPRJ2srMJW8ZmfbbFTAy7eG4o16Bf@52.10.46.160:3015">>, %% us-west-2
-        <<"aenode://pp$2vhFb3HtHd1S7ynbpbFnEdph1tnDXFSfu4NGtq46S2eM5HCdbC@18.195.109.60:3015">>, %% eu-central-1
-        <<"aenode://pp$27xmgQ4N1E3QwHyoutLtZsHW5DSW4zneQJ3CxT5JbUejxtFuAu@13.250.162.250:3015">>,%% ap-southeast-1
-        <<"aenode://pp$2i8N6XsjCGe1wkdMhDRs7t7xzijrjJDN4xA22RoNGCgt6ay9QB@31.13.249.70:3015">> %% eu-east
-      ]},
+  {aecore, [{beneficiary, <<"ak$tjnw1KcmnwfqXvhtGa9GRjanbHM3t6PmEWEWtNMM3ouvNKRu5">>},
+      {peers, []},
       {password, <<"secret">>},
       {db_path, "."},
       {persist, false},
```

Ok. (I do not like an extra file though I do not have a better idea without increasing complexity.)